spec/requests/api/v1/api_provides_client_with_collection_of_products_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+RSpec.describe Api::ProductsController, type: :request do
+  let!(:products) { 2.times { create(:product) } }
+  describe 'GET /product' do
+    before do
+      get '/api/products'
+    end
+
+    it 'should return a 200 response' do
+      expect(response).to have_http_status 200
+    end
+
+    it 'should return two products' do
+      expect(JSON.parse(response.body)['products'].count).to eq 2
```

Comment:
Please add the `response_json` module like you do in the Cooper challenge

Reply:
added

Reply:
Then please refactor this to `response_json` instead of `JSON.parse(response.body)` 🦖

Reply:
refactored